src/components/Datepicker/Datepicker.tsx

```diff
@@ -105,6 +107,8 @@ export const Datepicker: FC<DatepickerProps> = ({
   className,
   theme: customTheme = {},
   onSelectedDateChanged,
+  dateValue,
+  labelEmptyDate = 'No date selected',
```

Comment:
To that end, it seems like we could use the `placeholder` field instead of `labelEmptyDate`, too.

Reply:
That makes sense, will do.